helper/date_format.go

```diff
@@ -22,6 +22,28 @@ func DateFormat(s string) string {
 	return template.HTMLEscapeString(t.Format("02 January 2006"))
 }
 
+// TimeFormatHH extracts BST and GMT time value for 24hr clock from ISO8601 formatted timestamps
+func TimeFormatHH(s string) string {
+	t, err := time.Parse(time.RFC3339, s)
+	if err != nil {
+		log.Error(context.Background(), "failed to parse time", err)
+		return template.HTMLEscapeString(s)
+	}
+	t = localiseTime(&t)
+	return template.HTMLEscapeString(t.Format("15:04"))
+}
+
+// TimeFormathh extracts BST and GMT time value for 12hr clock from ISO8601 formatted timestamps
+func TimeFormathh(s string) string {
+	t, err := time.Parse(time.RFC3339, s)
+	if err != nil {
+		log.Error(context.Background(), "failed to parse time", err)
+		return template.HTMLEscapeString(s)
+	}
+	t = localiseTime(&t)
+	return template.HTMLEscapeString(t.Format("03:04pm"))
```

Comment:
This function body is identical to the 24h and a couple of other functions in this package. I would suggest creating a private function accepting the format string so to not have duplicated code

Reply:
this is done